src/components/tutorial-collection-cards/components/card-badges/index.tsx

```diff
@@ -16,23 +20,52 @@ const PRODUCT_ICON_MAP: Record<ProductOption, JSX.Element> = {
   vagrant: <ProductIcon productSlug="vagrant" />,
   waypoint: <ProductIcon productSlug="waypoint" />,
 }
-const CARD_BADGE_MAP: Record<CardBadgeOption, JSX.Element> = {
+/**
+ * Map all card badge options to icons
+ */
+const CARD_BADGE_ICON_MAP: Record<CardBadgeOption, JSX.Element> = {
   ...PRODUCT_ICON_MAP,
   video: <IconPlay16 />,
   interactive: <IconTerminalScreen16 />,
 }
 
+/**
+ * Map product badge options to badge labels
+ */
+const PRODUCT_LABEL_MAP: Record<ProductOption, string> = {
+  boundary: 'Boundary',
+  consul: 'Consul',
+  nomad: 'Nomad',
+  packer: 'Packer',
+  terraform: 'Terraform',
+  vault: 'Vault',
+  vagrant: 'Vagrant',
+  waypoint: 'Waypoint',
+}
+/**
+ * Map all card badge options to badge labels
+ */
+const CARD_BADGE_LABEL_MAP: Record<CardBadgeOption, string> = {
+  ...PRODUCT_LABEL_MAP,
+  video: 'Video',
+  interactive: 'Interactive',
+}
+
 function CardBadges({ badges }: CardBadgesProps) {
   return (
     <ul className={s.root}>
       {badges.map((badge: CardBadgeOption) => {
         return (
           <li key={badge}>
-            <Badge
-              ariaLabel={badge}
-              icon={CARD_BADGE_MAP[badge]}
-              className={s.badge}
-            />
+            <Tooltip label={CARD_BADGE_LABEL_MAP[badge]}>
+              <div role="button">
```

Comment:
🤔 Wondering about this `role`, since it's not technically interactive. But maybe because of the tooltip it is? Does this ensure that the screen reader goes to the badges? 

I'm more just curious here, I trust your judgement.

Reply:
Ah no great callout - this was a slip on my part, was having trouble getting tooltips working with `Badge` and forgot to clean this up 😅 Fixed now, this `div` & the `role` are gone ✅